UI/frontend-plugins/frontend-tools/output-timer.cpp

```diff
@@ -32,6 +32,18 @@ OutputTimer::OutputTimer(QWidget *parent)
 
 	recordingTimer = new QTimer(this);
 	recordingTimerDisplay = new QTimer(this);
+
+	QObject::connect(streamingTimer, &QTimer::timeout, this,
+			 &OutputTimer::EventStopStreaming);
+
```

Comment:
Please remove the newline here (and the ones below) to stay consistent with the connections further up.

Reply:
Done.